shared/lib_irradproc.cpp

```diff
@@ -1548,7 +1548,8 @@ void incidence(int mode, double tilt, double sazm, double rlim, double zen,
             }
             else if (en_backtrack) {
                 // TODO: add cross-axis slope angle parameter
-                double backtracking_rotation = backtrack(truetracking_rotation * 180 / M_PI, gcr);
+                double axis_slope = cross_axis_slope(slope_tilt, azm, slope_azm);
```

Comment:
`axis_slope` has potential to be misinterpreted as axis tilt if not reading carefully.  Maybe rename the function to `calc_cross_axis_slope` and then the variable can be named `cross_axis_slope`?  Just an idea :)

Also, I think the comment in the line above can be removed now.  Similar comments appear in this file in a couple other places.